script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_9_1.txt

```diff
@@ -137,62 +137,62 @@ The dogs head towards me.
 I can see countless lines on their bodies.
 <Page30>
 ―――But despite this,
-my paralyzed mind doesn't instruct me to fight nor run.
+my paralyzed mind doesn't instruct me to fight or to run.
 <Page31>
 The first of the two dogs leaps.
-Its speed is beyond that of a human.
+Its speed is incomparable to that of a human.
 The length of the 20-meter corridor was traversed in no more than two seconds.
 <Page32>
-The dog opens its mouth.
-Its fangs, which are aligned like saws, lunge towards my throat.
+The black dog opens its mouth.
+Its fangs, like rows of sawteeth, lunge for my throat.
 <Page33>
 Swift and precise.
 The moment I realize it's coming.
-The dog makes a gargling sound and its fangs pierce my throat.
+The dog's fangs sink into my throat with a wet squelch.
 I, Tohno Shiki, have died.
 <Page34>
 No. That can't be right.
-There's no way I would be killed from this, let alone die.
+There's no way I could be killed like this, no way I could die like this.
 People dying isn't enough to make me hesitate.
 ――――A hot summer day.
-A long time ago, or maybe it was seven years ago.
+A long time ago, or maybe it was seven years ago.//ずっと昔、それとも七年も前の話。
 At that time, I'm sure I saw a much more vividly gruesome scene than this――――
 <Page35>
 Stab.
-I plunge my knife into the head of the black dog that had bitten me on my neck.
+I plunge my knife into the forehead of the black dog that had snapped at the nape of my neck.
 The moment its fangs touched my skin, just before it bit into the carotid artery, my arm moved itself automatically.
 <Page36>
-I'm amazed even at myself.
-I wasted no time running the knife through the dog's forehead, as if I was a machine whose only function was to cut things.
+I'm amazed at myself.
+Like a machine whose only function is to slash and kill, I run my blade through the forehead of the dog with no wasted motion.
 Because that was where the first dog's 'line' was.
 <Page37>
-Under normal circumstances, an animal's movement will not immediately stop when its brain is damaged.
-Muscles that have already received signals from the brain will execute the last given command.
+Under normal circumstances, an animal's movement will not stop immediately when its brain is damaged.
+Muscles that have already received signals from the brain will execute the last given command for as long as they are able.
 <Page38>
 So even if I more or less tear its head apart, the dog's fangs will still bite my neck off.
 Yeah, well―――under normal circumstances, that is.
 <Page39>
 However, the dog is 'dead'.
-To die is to be stopped. It lost all of its effects the moment it was about to kill me.
+To die is to be stopped. At the moment it was about to kill me, it lost all validity.//あらゆる効力を失った
```

Comment:
`effects` and `validity` both still feel a bit off here. Expand to 'validity as a living thing'? Change fragment to `kill me, its existence simply ceased`? Better single word to fit the mood?

Reply:
I like 'validity' on its own